e2e/data/integration/mysqlapp.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
```

Comment:
are there any changes to this file from the original? I don't see any vars to replace. If not, could we avoid duplicating it?